### PR TITLE
get brands autusuggests search section 

### PR DESCRIPTION
**Pull request by Samaamro20**

### CODE REVIEW

File: Server/controllers/brands.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require('env2')('.env');
+
+exports.getBrands = (req, res) => {
+  let brands = [];
+  let brand={};
```
Comment (shahenazmonia) [inline Server/controllers/brands.js]: why did you define brands and brand? you didn't use it!

File: Server/controllers/brands.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+  var Airtable = require('airtable');
```
Comment (shahenazmonia) [inline Server/controllers/brands.js]: modules requirements should be out of the function

File: Server/controllers/brands.js (hunk 1 above)
Comment (Samaamro20) [reply Server/controllers/brands.js]: unuseful code, I well remove it

File: Server/controllers/brands.js (hunk 2 above)
Comment (Samaamro20) [reply Server/controllers/brands.js]: Done

**Comment by Samaamro20** [comment]
@shahenazmonia  Done :+1:

**Pull request merged**